### PR TITLE
refactor 328-logo-customization-functionality.md move LCM001 to DTS

### PR DESCRIPTION
**Pull request by Psotas**

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-compatibility/329-logo-customization.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Dasharo Compatibility: Logo customization
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Make yourself familiar with
```
Comment (miczyg1) [inline docs/unified-test-documentation/dasharo-compatibility/329-logo-customization.md:9]:
```suggestion
1. Get familiar with
```

File: docs/unified-test-documentation/dasharo-compatibility/329-logo-customization.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    [Logo customization procedure](/guides/logo-customization)
+
+## LCM001.001 Replace logo in existing image and flashing firmware
+
+**Test description**
+
+This test aims to verify whether replacing the logo in the existing image is
```
Comment (miczyg1) [inline docs/unified-test-documentation/dasharo-compatibility/329-logo-customization.md:16]:
```suggestion
The test aims to verify whether replacing the logo in the existing image is
```

File: docs/unified-test-documentation/dasharo-compatibility/329-logo-customization.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+will be shown properly.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
```
Comment (miczyg1) [inline docs/unified-test-documentation/dasharo-compatibility/329-logo-customization.md:27]:
```suggestion
   [Test cases common documentation](#test-cases-common-documentation) section.
```

File: docs/unified-test-documentation/dasharo-compatibility/329-logo-customization.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+    ```bash
+    chain http://boot.3mdeb.com/dts.ipxe
+    ```
+
+1. Wait for `Enter an option:`.
+1. Type in `9` and press `Enter`.
+1. Based on the
+    [dedicated documentation](/guides/logo-customization#build-image-with-custom-logo)
+    replace the logo in an existing image.
```
Comment (miczyg1) [inline docs/unified-test-documentation/dasharo-compatibility/329-logo-customization.md:52]:
```suggestion
   [dedicated documentation](/guides/logo-customization#build-image-with-custom-logo)
   replace the logo in an existing image.
```

File: mkdocs.yml [hunk 5]
```diff
@@ -380,6 +380,7 @@ nav:
               - 'Dasharo Tools Suite': unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md
               - 'Embedded controller flashing': unified-test-documentation/dasharo-compatibility/327-embedded_controller_flashing.md
               - 'Logo customization': unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md
```
Comment (miczyg1) [inline mkdocs.yml:382]: This file already has the logo replacement procedure:
`LCM001.001 Replace logo in existing image and flashing firmware`

Why do we write the same test again?

**Review by miczyg1 — CHANGES_REQUESTED** [review]
I think we should close the PR, it is doubling existing test case

File: mkdocs.yml (hunk 5 above)
Comment (Psotas) [reply mkdocs.yml:382]: So how to understand, I should rewrite the `LCM001.001 Replace logo in existing image and flashing firmware` test  to be under DTS, not ubuntu.
Should I not worry that the test setup requires more than necessary?
```
Test setup

    Proceed with the Generic test setup: firmware.
    Proceed with the Generic test setup: OS installer.
    Proceed with the Generic test setup: OS installation.
    Proceed with the Generic test setup: OS boot from disk.
    Make yourself familiar with Logo customization procedure
```
https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality/#test-cases-common-documentation

Comment (miczyg1) [reply mkdocs.yml:382]: We can change it to DTS, yes. The test setup may be stripped from unnecessary steps that involve installing Ubuntu etc.

Comment (Psotas) [reply mkdocs.yml]: I moved the test to [328-logo-customization-functionality](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality/#test-cases-common-documentation)

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md [hunk 6]
```diff
@@ -5,58 +5,51 @@
...
-    flashrom -p internal -w [path-to-binary] --ifd -i bios
+    chain http://boot.3mdeb.com/dts.ipxe
     ```
 
-1. Reboot DUT.
+1. Wait for `Enter an option:`.
+1. Type in `9` and press `Enter`.
+1. Based on the
+   [dedicated documentation](/guides/logo-customization#build-image-with-custom-logo)
+   replace the logo in an existing image.
```
Comment (miczyg1) [inline docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:53]:
```suggestion
1. Reboot the DUT and observe the boot logo.

```

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**